autoscheduler/frontend/src/components/Empty.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Is this temporary/will the prop ever be used so we can remove this line?

Reply:
Ah, I got it. Sending another commit